CronController.php

```diff
@@ -127,7 +127,7 @@ protected function transformDatePieces(array $parameters){
             array(0,6),  //Weekdays
         );
         foreach ($parameters AS $n => &$repeat) {
-            list($repeat, $every) = explode('\\', $repeat, 2) + array(false, 1);
+            list($repeat, $every) = explode('/', $repeat, 2) + array(false, 1);
```

Comment:
The `\\` syntax is probably leftover from the yii1 cron extension (http://www.yiiframework.com/extension/phpdoc-crontab/) which defined the cron timings inside PHP comments, the problem being, imagine you want to repeat every 2 mins, you would write `*/2` - but the `*/` would end the PHP comment, hence why `\` was used instead. 

I think with this yii2 extension, now that the timings are defined within the config file, we are safe to use the normal cron syntax.